.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Issues 
+
+<!--- 
+Add the following statement necessarily: 
+
+Fixes #ISSUE_NUMBER 
+-->
+
+Fixes #
+
+## Checklist 
+
+- [ ] The Guide has been linted against a language and grammar tool.
+- [ ] The Guide is easy to follow up with and understand for new users.
```

Comment:
Do you mean this "Guide" is easy to follow and understand for new users? Or that new users are following up with this guide afterwards? If it's the former, I suggest something like "The Guide is easy to follow and understand for new users."